Casbin/Model/DefaultSections.cs

```diff
@@ -85,12 +85,13 @@ public bool AddSection(string section, string key, string value)
         Dictionary<string, IReadOnlyAssertion> assertionMap = new() { [key] = assertion };
 
         // TryAdd is not supported in .NET 4.5
-        if (_assertionsMap.ContainsKey(key))
+        if (_assertionsMap.ContainsKey(key) ||
```

Comment:
Why not use `_assertionsMap.ContainsKey(section)`

Reply:
@Tanyuu reply to all review comments and resolve if needed